packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -213,14 +209,14 @@ function getTypeNamePermissive(fieldType: string, resolvedType: Protobuf.Type |
         throw new Error('Found field with no usable type');
       }
       const typeInterfaceName = getTypeInterfaceName(resolvedType);
-      if (resolvedType instanceof Protobuf.Type) {
+      if (resolvedType instanceof Protobuf.Type || resolvedType instanceof Protobuf.Enum) {
         if (repeated || map) {
           return inputName(typeInterfaceName);
         } else {
           return `${inputName(typeInterfaceName)} | null`;
```

Comment:
Does Protobuf.js accept `null` as an input for an enum field?

Reply:
Good catch. fixed to not allow `null` and match the previous types.